src/components/TodoModal/TodoModal.tsx

```diff
@@ -2,11 +1,41 @@
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+
+type Props = {
+  selectTodo: Todo;
```

Comment:
Consider renaming `selectTodo` to `selectedTodo` to better reflect its purpose and align with naming conventions.